examples/index.html

```diff
@@ -0,0 +1,225 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8" />
+  <title>Smart Panel Demo</title>
+  <meta name="description" content="Demo app for smart-scrollbox" />
+  <link rel="stylesheet" type="text/css" href="../bower_components/smart-button/style.css">
```

Comment:
Since your example depends on smart button, you should write a doc to tell others to install smart-button at root folder.